packages/component/src/providers/internal/SendBox/SendBoxComposer.tsx

```diff
@@ -116,10 +132,33 @@ const SendBoxComposer = ({ children }: PropsWithChildren<{}>) => {
         ]) as readonly [Timeout, Timeout];
       } else {
         scrollToEndRef.current?.();
-        apiSubmitSendBox();
+        if (combineAttachmentsAndText && files.length) {
```

Comment:
Few lines above, it already checked if there are any errors. The check here (`files.length`) is meaningless.